.swcrc

```diff
@@ -2,17 +2,10 @@
   "minify": false,
   "jsc": {
     "parser": {
-      "syntax": "ecmascript",
-      "jsx": true,
-      "dynamicImport": false,
-      "privateMethod": false,
-      "functionBind": false,
-      "exportDefaultFrom": false,
-      "exportNamespaceFrom": false,
+      "syntax": "typescript",
```

Comment:
We should make a follow up diff to move this config under the `bazel/js/internal` folder. I was being quick and dirty when I added it.

Reply:
I actually tried this, SWC was complaining when I tried to point it to a json file via `$(rootpath ...)`, but I'm sure it's doable